composer.json

```diff
@@ -12,14 +12,14 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=7.1",
-        "composer-plugin-api": "^1.0|^2.0"
+        "composer-plugin-api": ">=1.0"
```

Comment:
Do we actually want to do this ? This seems dangerous regarding changes in the composer plugin API (and using an incompatible plugin in composer can break you ability to upgrade...)

Reply:
ok, reverting, we might decide to widen later on if needed

Reply:
Reverted in https://github.com/symfony/flex/commit/e2ea724773cc7c391e0d163967675acd31ee04c7